README.md

```diff
@@ -49,37 +49,43 @@ User can add a STM32 based board following this [wiki](https://github.com/stm32d
 
 ## Supported boards
 
- - [Nucleo 144 boards](#nucleo-144-boards)
- - [Nucleo 64 boards](#nucleo-64-boards)
- - [Nucleo 32 boards](#nucleo-32-boards)
- - [Discovery boards](#discovery-boards)
- - [Eval boards](#eval-boards)
- - [STM32MP1 series coprocessor boards](#stm32mp1-series-coprocessor-boards)
- - [Generic STM32C0 boards](#generic-stm32c0-boards)
- - [Generic STM32F0 boards](#generic-stm32f0-boards)
- - [Generic STM32F1 boards](#generic-stm32f1-boards)
- - [Generic STM32F2 boards](#generic-stm32f2-boards)
- - [Generic STM32F3 boards](#generic-stm32f3-boards)
- - [Generic STM32F4 boards](#generic-stm32f4-boards)
- - [Generic STM32F7 boards](#generic-stm32f7-boards)
- - [Generic STM32G0 boards](#generic-stm32g0-boards)
- - [Generic STM32G4 boards](#generic-stm32g4-boards)
- - [Generic STM32H7 boards](#generic-stm32h7-boards)
- - [Generic STM32L0 boards](#generic-stm32l0-boards)
- - [Generic STM32L1 boards](#generic-stm32l1-boards)
- - [Generic STM32L4 boards](#generic-stm32l4-boards)
- - [Generic STM32L5 boards](#generic-stm32l5-boards)
- - [Generic STM32U5 boards](#generic-stm32u5-boards)
- - [Generic STM32WB boards](#generic-stm32wb-boards)
- - [Generic STM32WL boards](#generic-stm32wb-boards)
- - [3D printer boards](#3d-printer-boards)
- - [Blues Wireless boards](#blues-wireless-boards)
- - [Elecgator boards](#elecgator-boards)
- - [Electronic Speed Controller boards](#electronic-speed-controller-boards)
- - [Garatronics boards](#Garatronic/McHobby-boards)
- - [Generic flight controllers](#generic-flight-controllers)
- - [LoRa boards](#lora-boards)
- - [Midatronics boards](#midatronics-boards)
+- [Arduino core support for STM32 based boards](#arduino-core-support-for-stm32-based-boards)
```

Comment:
Here we are in the Supported boards § so other link should be removed: introduction,....